README.md

```diff
@@ -293,12 +293,12 @@ You can use the following options at build time to influence the build. During t
 |png|`--with-png`|Whether to build with libpng support|Auto|
 |gcov|`--enable-gcov`|Build gcov coverage tests|No|
 |tests|`--enable-tests`|Build tests (requires `bash`)|No|
-|python2|`--enable-python`|Build Python library integration|No|
+|python3|`--enable-python`|Build Python library integration|No|
```

Comment:
User software may consist of legacy Python 2 scripts. It would be better to add Python 3 support like this:

* if `--enable-python` ⇒ enable python3, give warning that this used to enable python2 and that function is now behind `--enable-python2`
* if `--enable-python2` ⇒ enable python2
* if `--enable-python3` ⇒ enable python3 (explicit version)

Reply:
i know that most distros are not allowing python2 packages, though they are generally still shipping python2 interpreters. i don't want to put any effort, going forward, into maintaining python2 code -- legacy python2 scripts ought by now have been updated to python3. python2 was EOLd over a year ago.

i'm very hesitant to continue shipping python2. if someone truly needs it, they can grab an old release, no?

wouldn't the python2 scripts need to have an explicit `/usr/bin/env python2` or whatever to work on most distros? are we proposing to rewrite the shebang lines at installation time?